### DISCUSSION TITLE
the YZ plane pole figure

### DISCUSSION THREAD

**Discussion by m-amrei** [mtex-toolbox/mtex#1694] [Ask Anything]
Hello, 
I would like to have the pole figures in YZ plane instead of the default XY plane. I am aware of the rotation tricks but I would like to know if there is a better solution instead of touching the source data.
Regards, 
Mohsen

**Comment by Tijmenvermeij** [comment]
I'm not sure where your current X and Y directions are pointing to, but you can try playing with the plotting directions in Mtex.
If you type
`getMTEXpref`
you can check the current plotting directions, for example they could be
```
xAxisDirection: 'east'
zAxisDirection: 'intoPlane'
```

you can now change the plotting, for example for x:
`setMTEXpref('xAxisDirection','north');`
etc. until you get what you want.
